django_airavata/apps/groups/forms.py

```diff
@@ -0,0 +1,31 @@
+# Create your forms here.
+from django import forms
+
+class CreateForm(forms.Form):
+    domain_id = forms.CharField(required=True, disabled=True)
```

Comment:
How can I mention the widget keyword twice? I mean, we need _widget=forms.RadioSelect_ also, right?

Reply:
No, for the group_type and group_cardinality, just make the widget=forms.HiddenInput. You also don't need to specify choices for them.

The widget affects how the form is rendered. With HiddenInput it will render the form field as
```
<input type="hidden" name="..." value="...">
```